Update Travel_fellowships.md with the rubrics for review (#89)
OBF/obf-docs: Travel_fellowships.md

### Promoting diversity and representation

> One of the goals of the OBF fellowship is to enable and enhance active participation from groups otherwise underrepresented at the event or in the broader community, including but not limited to underrepresented country of residence or origin, ethnic background (historically underrepresented and other minority groups), career stages, sexuality, gender identity and expression, and people with disabilities.

Reviewer: I like the rephrasing.

> We ask applicants to describe how their participation in this event help promote diversity and representation of members from marginalised and traditionally underrepresented groups in the _overall_ open source bioinformatics and/or open science community.

Reviewer 2: Isn't the noun here (_their participation_) singular, not plural? Here, this should be _helps_, no?

> We ask applicants to describe how their participation in this event help promote diversity and representation of members from marginalised and traditionally underrepresented groups in the _overall_ open source bioinformatics and/or open science community.

Reviewer 3: I agree--that's what I was suggesting, that it should be "helps", not "help" (as it was erroneously changed to).

> We ask applicants to describe how their participation in this event help promote diversity and representation of members from marginalised and traditionally underrepresented groups in the _overall_ open source bioinformatics and/or open science community.

Reviewer 2: Ah I see. Sorry I got mixed up.

> We ask applicants to describe how their participation in this event help promote diversity and representation of members from marginalised and traditionally underrepresented groups in the _overall_ open source bioinformatics and/or open science community.

Author: ```suggestion
We ask applicants to describe how their participation in this event helps promote diversity and representation of members from marginalised and traditionally underrepresented groups in the _overall_ open source bioinformatics and/or open science community.
```

The selection of the fellowship awardees is based on the criteria described below. 
We expect applicants to include sufficient information in their application to allow the review panel to assess their eligibility fairly.

#### Event and nature of participation

The review panel will assess how the event that the applicants have selected and the nature of their participation promote open science practices in bioinformatics and/or biological research community.
It's important to keep in mind that the event should be scheduled to occur anytime in a one-year window starting one month after the application deadline.

> Events that are eligible may include but are not limited to conferences, workshops, code fests, hackathons, training courses, collaborative sprints, informal meet-ups or other skill-building and networking event. 

Reviewer: Although we fund all these events, the majority still consider the funding to be aimed at attending conferences. I think we can get more applications from skill-building events (workshops, training courses, etc). Let's find ways to highlight the kind of events they can apply to (maybe through a blog-post)...

Author: Thank you for adding this in the blog.

#### Promoting diversity and representation

The review panel will look at how applicants' attendance will contribute to increasing participation from diverse members at the event.

> We are aware that members from many marginalised groups are still underrepresented in the bioinformatics, computational fields or tech community more broadly.

Author: ```suggestion
We are aware that members from many marginalised groups are still underrepresented in bioinformatics, computational fields or the tech community more broadly.
```

> Our efforts with this fellowship are to support members from underrepresented demographic groups including but not limited to diverse ethnic backgrounds, career stages, sexuality, gender identity and expression, people with disabilities and members from low-income backgrounds (such as from developing nations).

Author: ```suggestion
Our goals with this fellowship are to support members from underrepresented demographic groups including but not limited to diverse ethnic backgrounds, career stages, sexuality, gender identity and expression, people with disabilities and members from low-income backgrounds (such as from developing nations).
```

The review panel will look at how applicants' attendance will contribute to increasing participation from diverse members at the event.
We are aware that members from many marginalised groups are still underrepresented in the bioinformatics, computational fields or tech community more broadly.
Our efforts with this fellowship are to support members from underrepresented demographic groups including but not limited to diverse ethnic backgrounds, career stages, sexuality, gender identity and expression, people with disabilities and members from low-income backgrounds (such as from developing nations).

If applicants are not from a minority or underrepresented group but have a track record for advocating for diversity as an ally through their work or involvement in an event, they are welcome to apply for this fellowship as well.

*We do not consider the diversity of operating systems, programming languages or scientific domains as a criterion for the OBF Event Fellowship.*

## Rubric for Application Review

We ask reviewers to evaluate the application across multiple dimensions with scores of 1, 2, or 3 that they share via a Google form with the Event Fellowship chair. 

> These correspond generally to reviews of "criteria not met", "criteria partially met" and "criteria fully met":

Reviewer 4: Maybe. I thought multiple scores, so plural? i.e. criteria (as it stands).

> These correspond generally to reviews of "criteria not met", "criteria partially met" and "criteria fully met":

Author: This is an overview sentences to represent general scoring through multiple scores. I will leave it as criteria.

- Applicants who score mostly 1s do not have a clear understanding of the overall goals of this fellowship to support Open Source software practices and diversity of attendees in bioinformatics/biological research/open science events and communities.

> - Applicants who score mostly 2s are enthusiastic if not wholly aligned with the event fellowship's goals, for example, they may not be attending an open-source related event but have an interest in learning and promoting best practices, or that they want to attend a suitable event but do not represent any underrepresented minority groups.

Reviewer 4: Yep, simpler.

> - Applicants who score mostly 2s are enthusiastic if not wholly aligned with the event fellowship's goals, for example, they may not be attending an open-source related event but have an interest in learning and promoting best practices, or that they want to attend a suitable event but do not represent any underrepresented minority groups.

Author: ```suggestion
- Applicants who score mostly 2s are enthusiastic if not wholly aligned with the event fellowship's goals, for example, they may not be attending an open-source related event but have an interest in learning and promoting best practices, or they want to attend a suitable event but do not represent any underrepresented minority groups.
```

> - Applicants who score mostly 3s are ready, goal-oriented, interested in contributing to the open-source, and represent traditionally underrepresented or minority groups in open science/bioinformatics.

Author: ```suggestion
- Applicants who score mostly 3s are ready, goal-oriented, interested in contributing to open-source, and represent traditionally underrepresented or minority groups in open science/bioinformatics.
```

Reviewers are also asked whether they would recommend an application. The available answers are "No", "Yes", "Unsure" or "Request to discuss". Candidates who have ratings of "No" or "Unsure" across all reviewers will be rejected. All other candidates will be discussed at the panel selection.

We ask all reviewers to finish the review with a few summary sentences to describe the application and the motivations for their scores to facilitate conversation during the selection panel discussion.
Below we provide a copy of our review form.

### Event and nature of applicant's participation

> 

Author: I will update them below with 'criterion' (for individual scoring) and leave the overview section with 'criteria' ro reflect multiple scores

> - [ ] 1 (criteria not met) Does not provide enough information or seems to misunderstand what kind of event OBF Event fellowship supports
> - [ ] 2 (criteria partially met) The event or the nature of their participation may not directly relate to Open Source in bioinformatics, but they have some understanding/plan on how they would use the knowledge gained at this event to promote open source practices and diversity
> - [ ] 3 (criteria fully met) Seems to have a clear understanding of how their participation in this event relates to open-source bioinformatics or open science in general, and how they will further use and promote the knowledge obtained at this event

Author: ```suggestion
- [ ] 1 (criterion not met) Does not provide enough information or seems to misunderstand what kind of event OBF Event fellowship supports
- [ ] 2 (criterion partially met) The event or the nature of their participation may not directly relate to Open Source in bioinformatics, but they have some understanding/plan on how they would use the knowledge gained at this event to promote open source practices and diversity
- [ ] 3 (criterion fully met) Seems to have a clear understanding of how their participation in this event relates to open-source bioinformatics or open science in general, and how they will further use and promote the knowledge obtained at this event
```

### Open Source Resources

> - [ ] 1 (criteria not met) Shares none or unrelated/irrelevant information on open source resource and how they relate to their proposed event

Reviewer 4: --> information on open source resource**s** (add trailing s)

Author: ```suggestion
- [ ] 1 (criteria not met) Shares none or unrelated/irrelevant information on open source resources and how they relate to their proposed event
```

> - [ ] 2 (criteria partially met) Shares some relevant information on open source resource and how they relate to their proposed event
> - [ ] 3 (criteria fully met) Shares appropriate and relevant information on open source resource and how they relate to their proposed event

Author: ```suggestion
- [ ] 2 (criterion partially met) Shares some relevant information on open source resources and how they relate to their proposed event
- [ ] 3 (criterion fully met) Shares appropriate and relevant information on open source resources and how they relate to their proposed event
```

> ### Promoting diversity and representation of historically excluded or underrepresented groups *

Reviewer 4: Why the * here?

Author: ```suggestion
### Promoting diversity and representation of historically excluded or underrepresented groups
```

Author: That was copy paste error (Google form uses * for required section)

> - [ ] 1 (criteria not met) The application lacks information or any justification on how their participation will help promote diversity and representation
> - [ ] 2 (criteria partially met) The application provides little information or some justification on how their participation will help promote diversity and representation
> - [ ] 3 (criteria fully met) The application provides sufficient information or justification on how their participation will help promote diversity and representation
> 
> ### Justification for funding
> - [ ] 1 (criteria not met) The requested funding does not fit our criteria of qualified items that can be funded through this fellowship and the applicant has not provided enough justification
> - [ ] 2 (criteria partially met) The requested funding partially fits our criteria of qualified items that can be funded through this fellowship and the applicant has provided enough justification
> - [ ] 3 (criteria fully met) The requested funding completely fits our criteria of qualified items that can be funded through this fellowship and the applicant has provided sufficient justification

Author: ```suggestion
- [ ] 1 (criterion not met) The application lacks information or any justification on how their participation will help promote diversity and representation
- [ ] 2 (criterion partially met) The application provides little information or some justification on how their participation will help promote diversity and representation
- [ ] 3 (criterion fully met) The application provides sufficient information or justification on how their participation will help promote diversity and representation

### Justification for funding
- [ ] 1 (criterion not met) The requested funding does not fit our criteria of qualified items that can be funded through this fellowship and the applicant has not provided enough justification
- [ ] 2 (criterion partially met) The requested funding partially fits our criteria of qualified items that can be funded through this fellowship and the applicant has provided enough justification
- [ ] 3 (criterion fully met) The requested funding completely fits our criteria of qualified items that can be funded through this fellowship and the applicant has provided sufficient justification
```

> ### Would you recommend this applicant for OBF Event Fellowship? *

Author: ```suggestion
### Would you recommend this applicant for OBF Event Fellowship?
```

- [ ] No
- [ ] Yes

> - [ ] Unsure/Request to discuss

Reviewer 4: Why the * here?

Reviewer 3: i think that may be a relic caused by copying the questions from the review form. required (non-optional) questions are marked there with *.

Reviewer 4: Ah. Maybe added [required] to the text, or just drop the asterisk?

### Please provide 1-2 summary sentences about this application to facilitate the selection. Please include what this application is about, what its open source aspect is, how does it promote diversity and representation and what funding would you recommend to cover.
- [Open text response]

### Please highlight any diversity/minority groups this applicant belongs to, which should be supported for their representation in open source bioinformatics and biological research fields
- [Open text response]

### Anything else?

- [Open text response]

> 

Reviewer: The Rubric is great, thanks Malvika.
